test/performance/exit-code.perfkeys

```diff
@@ -0,0 +1 @@
+key:
```

Comment:
I don't think we can actually commit this test, since I think we want this to be an error, so this test would always fail. Useful to test manually, but I don't see an easy way to run this automatically.